Library/Formula/tesseract.rb

```diff
@@ -128,6 +128,10 @@ class Tesseract < Formula
     sha1 "a559801a42be8a3dc15ae2ec27da99bd8b0b85ac"
   end
 
+  resource "tessdata-head" do
+    url 'https://github.com/tesseract-ocr/tessdata.git'
```

Comment:
And double quotes around here. Everything should be wrapped in double quotes, we've been moving across more strictly to the Ruby style guide, but new lines in particular shouldn't reinforce the single quotes :)